modules/data-entry/src/main/frontend/src/questionnaire/Form.jsx

```diff
@@ -488,6 +488,22 @@ function Form (props) {
     return time.hasSame(DateTime.local(),"day") ? "at " + time.toFormat("hh:mma") : time.toRelativeCalendar();
   }
 
+  let validLinks = data?.["cards:links"]?.filter(link => link["to"]?.startsWith("/"));
+  let links = validLinks.length > 0 ?
+      (
+        <Typography variant="overline">
+          {"Related: "}
+          {validLinks.length == 1 ?
+              validLinks.map(link => <Link key={link["@name"]} to={"/content.html" + link["to"]}>{link["resourceLabel"]}</Link>)
+              :
+              <List>
```

Comment:
The fastest way to reduce padding without redesigning the UI is to make the list `dense` and maybe also `disablePadding`.
```suggestion
              <List dense disablePadding>
```